packages/fxa-shared/feature-flags/index.js

```diff
@@ -28,22 +28,25 @@ module.exports = initialise;
  *
  * @returns {FeatureFlags}
  */
-function initialise (config, log, defaults) {
+function initialise(config, log, defaults) {
   const { interval, redis: redisConfig } = config;
 
-  if (! (interval > 0 && interval < Infinity)) {
+  if (!(interval > 0 && interval < Infinity)) {
     throw new TypeError('Invalid interval');
   }
 
-  if (! log) {
+  if (!log) {
```

Comment:
I'll be sorry to see these go fwiw. I was converted to whitespace-after-`!` when I joined Mozilla and have since been of the opinion that it genuinely improves readability. Is it configurable, or a necessary casualty of war?

Reply:
> or a necessary casualty of war?

[Prettier's options list](https://prettier.io/docs/en/options.html) is intentionally very limited, this doesn't seem to be one of the options.